tests/contrib/dd_trace_api/test_integration.py

```diff
@@ -0,0 +1,107 @@
+import sys
+from typing import Any
+
+import dd_trace_api
+
+from ddtrace._trace.span import Span as dd_span_class
+from ddtrace.contrib.internal.dd_trace_api.patch import patch
+from ddtrace.contrib.internal.dd_trace_api.patch import unpatch
+from tests.utils import TracerTestCase
+
+
+class DDTraceAPITestCase(TracerTestCase):
```

Comment:
should we do snapshot tests instead?

would be a really good validation that importing and using the dd_trace_api package produces spans that we expect.